internal/models/kubernetessecret/cluster/secret_type.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Please move the contents of this file to the spec file itself since it is a part of the spec.

Reply:
Updated PTAL